extend eslint:recommended is broken

```
/Users/sboudrias/github/tmp/generator-demo/node_modules/gulp-eslint/node_modules/eslint/lib/config.js:129
                    throw e;
                          ^
Error: Cannot find module 'eslint-config-eslint:recommended'
Referenced from: /Users/sboudrias/github/tmp/generator-demo/.eslintrc
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at loadConfig (/Users/sboudrias/github/tmp/generator-demo/node_modules/gulp-eslint/node_modules/eslint/lib/config.js:96:48)
    at /Users/sboudrias/github/tmp/generator-demo/node_modules/gulp-eslint/node_modules/eslint/lib/config.js:123:46
    at Array.reduceRight (native)
    at loadConfig (/Users/sboudrias/github/tmp/generator-demo/node_modules/gulp-eslint/node_modules/eslint/lib/config.js:111:36)
    at getLocalConfig (/Users/sboudrias/github/tmp/generator-demo/node_modules/gulp-eslint/node_modules/eslint/lib/config.js:225:23)
    at Config.getConfig (/Users/sboudrias/github/tmp/generator-demo/node_modules/gulp-eslint/node_modules/eslint/lib/config.js:363:22)
```

Looks like we need the config somewhere if we're going to extend it.

cc @zckrs

--- a/generators/gulp/index.js
+++ b/generators/gulp/index.js
@@ -34,7 +34,7 @@ module.exports = generators.Base.extend({
       extend(pkg, {
         devDependencies: {
           gulp: '^3.6.0',
-          'gulp-eslint': '^0.15.0',
+          'gulp-eslint': '^1.0.0',
           'gulp-exclude-gitignore': '^1.0.0',
           'gulp-istanbul': '^0.9.0',
           'gulp-mocha': '^2.0.0',

